brainstormforce/header-footer-elementor
# Handled deprecation of `_content_template()`.

### Description
_content_template() deprecations.
PHP Deprecated: _content_template is deprecated since version 2.9.0! Use Elementor\Controls_Stack::content_template() instead.

### Screenshots


### Types of changes
- Handled deprecation.





### How has this been tested?
1. Check functionality of content template of all widgets.
2. No _content_template() related deprecation notice in the debug.log
3. Check the same functionality on Elementor 3.0.0 version.

### Checklist:
